scss/components/_icons.scss

```diff
@@ -12,9 +12,9 @@
 
   .badge {
     box-shadow: $box-shadow-lg;
-    left: px(-7);
+    left: -.5em;
     position: relative;
-    top: px(-7);
+    top: -.5em;
```

Comment:
Shouldn't these be rems/px? In this case, the most relevant nearest parent (the icon) has no font size specified thus the behavior is unpredictable.